index.js

```diff
@@ -16,12 +17,16 @@ function checkForDefaults(defaults) {
  * @param {Config} defaults
  * @param {String} [configFilename]
  */
-module.exports = (robot, defaults, configFilename = 'remind-merge.yml') => {
+module.exports = (robot, defaults = {merged: "You've merged!", unmerged: "You didn't merge."}, configFilename = 'remind-merge.yml') => {
```

Comment:
@hectorsector does "You didn't merge." get displayed or does "It looks like you closed this Pull Request without merging. If you need any help, just ask!" get displayed for the user? If "You didn't merge" gets displayed we might want to make it warmer.

Reply:
👍 these are defaults if the person doesn't include a configuration file. I guess I'm not the warmest person, huh? 😝 Feel free to edit the default text directly, @beardofedu.